widgetry/src/tools/colors.rs

```diff
@@ -38,10 +38,13 @@ impl ColorLegend {
                     .map(|(idx, color)| ((idx as f64) / ((n - 1) as f64), *color))
                     .collect(),
             }),
-            Polygon::union_all(
+            Tessellation::union_all(
```

Comment:
This is one of the real reasons why unioning tessellations is actually necessary. This is a bunch of rectangles in a row. Why not just one big rectangle? Because we need the points in the middle in order for the shader logic to actually have a chance to evaluate here
![Screenshot from 2022-08-30 15-54-59](https://user-images.githubusercontent.com/1664407/187470131-f7b1e367-530b-4e3a-8d32-7c3d8868527e.png)